### PR TITLE
Add plane aabb intersection

### PR DESCRIPTION
**Pull request by rokalax**
Add plane aabb intersection along with some test scenarios

### CODE REVIEW

File: pyrr/geometric_tests.py
Comment (adamlwgriffiths) [inline pyrr/geometric_tests.py:1]: These tests are failing because the 'pyrr' module isn't imported here. These tests use the submodules directly.

**Comment by adamlwgriffiths** [comment]
Thanks for adding this, and for writing the tests.
There are currently some import issues, if you could please fix that and I'll re-run the checks.

File: pyrr/geometric_tests.py
Comment (rokalax) [reply pyrr/geometric_tests.py:1]: Hello, sure, my bad.  I will fix this asap. Thanks

Comment (rokalax) [reply pyrr/geometric_tests.py:1]: Hello Adam, I should have fixed the issues.

**Pull request closed without merging**